src/applications/static-pages/createCallToActionWidget.js

```diff
@@ -2,20 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 
-export default function createCallToActionWidget(store) {
+export default async function createCallToActionWidget(store) {
   const widgets = Array.from(document.querySelectorAll('.cta-widget'));
 
   if (widgets.length) {
-    widgets.forEach(async el => {
-      await import('../../platform/site-wide/cta-widget/sass/cta-widget.scss');
-
-      const {
-        default: CallToActionWidget,
-      } = await import('../../platform/site-wide/cta-widget');
+    const {
+      default: CallToActionWidget,
+    } = await import(/* webpackChunkName: "cta-widget" */ '../../platform/site-wide/cta-widget');
 
+    widgets.forEach((el, index) => {
       ReactDOM.render(
         <Provider store={store}>
-          <CallToActionWidget appId={el.dataset.appId} />
+          <CallToActionWidget appId={el.dataset.appId} index={index} />
```

Comment:
Added index in anticipation of pages that have two CTAs with different content, actions, etc.